core/src/test/java/com/sequenceiq/distrox/v1/distrox/service/upgrade/ComponentLockerTest.java

```diff
@@ -0,0 +1,99 @@
+package com.sequenceiq.distrox.v1.distrox.service.upgrade;
+
+import static com.sequenceiq.distrox.api.v1.distrox.model.upgrade.DistroxUpgradeShowAvailableImages.LATEST_ONLY;
+import static com.sequenceiq.distrox.api.v1.distrox.model.upgrade.DistroxUpgradeShowAvailableImages.SHOW;
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import com.sequenceiq.distrox.api.v1.distrox.model.upgrade.DistroxUpgradeShowAvailableImages;
+import com.sequenceiq.distrox.api.v1.distrox.model.upgrade.DistroxUpgradeV1Request;
+
+@ExtendWith(MockitoExtension.class)
+class ComponentLockerTest {
+
+    private static final String USER_CRN = "userCrn";
```

Comment:
this could be local variable in method

Reply:
could be, but I prefer this way, less line in the method and it's obvious this way it a constant. Also if we add a new method we could reuse that way.